project_task_description_template/__manifest__.py

```diff
@@ -0,0 +1,22 @@
+# Copyright (C) 2023 Cetmix OÜ
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/lgpl)
+
+{
+    "name": "Project Task Templates",
+    "version": "14.0.1.0.0",
+    "summary": "Module for Task Templates",
+    "author": "Cetmix, Odoo Community Association (OCA)",
+    "license": "AGPL-3",
+    "category": "Project",
+    "maintainers": ["dessanhemrayev", "CetmixGitDrone"],
+    "website": "https://github.com/OCA/project",
+    "depends": ["project"],
+    "demo": [
+        "data/demo.xml",
+    ],
+    "data": [
+        "views/task_templates.xml",
+        "security/ir.model.access.csv",
```

Comment:
Security must be located in first position in data section

Reply:
okay, I'll fix it